files/WinesapDualboot.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Exec=/bin/bash /home/winesap/.winesapos/WinesapDualboot.sh
+Name=winesapOS Dualboot Beta Setup
+Comment=winesapOS setup but for dualbooting
```

Comment:
nit: "dualbooting" should be "dual-booting" (when it is a verb, it has the "-" hyphen).

The `Comment=` can actually be removed as it is redundant with the `Name=`.